src/main/python/tensorframes/core.py

```diff
@@ -37,6 +37,19 @@ def _get_shape(node):
     l = node.get_shape().as_list()
     return [-1 if x is None else x for x in l]
 
+def _initialize_variable(graph, fetches, initial_variable):
+    with tf.Session(graph=graph) as sess:
```

Comment:
My suggestion was simply to list all the variables in the graphs. If the list is empty, simply return the graph.